13-TeV-examples/uproot_python/HZZAnalysis.ipynb

```diff
@@ -110,15 +110,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 13,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import sys\n",
+    "#import sys\n",
     "# update the pip package installer\n",
-    "!{sys.executable} -m pip install --upgrade --user pip\n",
+    "#!{sys.executable} -m pip install --upgrade --user pip\n",
     "# install required packages\n",
-    "!{sys.executable} -m pip install --upgrade --user uproot==5.3.9 awkward==2.6.6 vector==1.4.1 numpy==2.0.0 matplotlib==3.9.0 aiohttp==3.9.5 requests==2.32.3"
+    "#!{sys.executable} -m pip install --upgrade --user uproot==5.3.9 awkward==2.6.6 vector==1.4.1 numpy==2.0.0 matplotlib==3.9.0 aiohttp==3.9.5 requests==2.32.3"
```

Comment:
Why did we comment these lines? If someone does not use binder, I guess they are needed? If they are not needed at all, we can also delete the paragraph.